plugins/single_plugins/idle.cpp

```diff
@@ -242,7 +242,8 @@ class wayfire_idle_plugin : public wf::per_output_plugin_instance_t
             return;
         }
 
-        if (!timeout_screensaver.is_connected() && (state == CUBE_SCREENSAVER_RUNNING))
+        if (!timeout_screensaver.is_connected() &&
```

Comment:
Should we maybe only call uninhibit_output if the screensaver is not running?

Reply:
I pushed a change, can you have another look?